src/coreclr/vm/gcinfodecoder.cpp

```diff
@@ -2140,6 +2144,9 @@ template <typename GcInfoEncoding> OBJECTREF* TGcInfoDecoder<GcInfoEncoding>::Ge
     }
     else if( GC_CALLER_SP_REL == spBase )
     {
+#ifndef FEATURE_NATIVEAOT
+        _ASSERTE(pRD->IsCallerContextValid);
+#endif
```

Comment:
```suggestion
```
`GET_CALLER_SP` does this validation, so we do not need to repeat it here. Also, I suspect that this would be a build break for #113888.

Reply:
I have added this validation before adding that function and somehow missed this was the only place where the `GET_CALLER_SP` was used. I'll remove it.